randovania/games/planets_zebeth/logic_database/Brinstar.txt

```diff
@@ -493,12 +487,16 @@ Extra - map_name: Brinstar/x392_y37
 > Door to Corridor-connecting room; Heals? False
   * Layers: default
   * Normal Door to Corridor-connecting room/Door to Corridor 7
+  > Door to End shaft
+      Knowledge (Beginner) and Can Use Bombs
```

Comment:
as this is a knowledge trick, please put a comment explaining what the knowledge is for